Felienne/hedy
# Always enables localized syntax highlighting if available

**Description**
In this PR we reverse a change made long ago where we only highlight localized syntax when logged in. This due to possible over-coloring and confusion. We reverse this change and always allow localized coloring if there are keywords for the selected language (also when not logged in).

**Fixes**
This PR fixes #3133

**How to test**
Make sure you are **not** logged in. Select any non-english language that supports localized keywords. Verify that these localized keywords are also highlighted in the code editor.
